docs/markdown.md

```diff
@@ -1,7 +1,7 @@
 # Markdown Scripts
 
 It's possible to write scripts using markdown. Only code blocks will be executed
-by zx. 
+by zx. Refer [Fenced Code blocks](https://spec.commonmark.org/0.30/#fenced-code-blocks)
```

Comment:
We are definitely not following commonmark. Let's remove this remark.

Reply:
Okay. I removed it!